Cargo.toml

```diff
@@ -1,16 +1,16 @@
 [package]
 
-name = "rocksdb"
+name = "parity-rocksdb"
 description = "A Rust wrapper for Facebook's RocksDB embeddable database."
-version = "0.4.5"
-authors = ["Tyler Neely <t@jujit.su>", "David Greenberg <dsg123456789@gmail.com>"]
+version = "0.5.0"
+authors = ["Parity Technologies <admin@parity.io>"]
```

Comment:
I'd probably keep the original authors in there too, like `authors = ["Tyler Neely <t@jujit.su>", "David Greenberg <dsg123456789@gmail.com>, "Parity Technologies <admin@parity.io>"]`